DynamoDB/CreateTable.js

```diff
@@ -0,0 +1,70 @@
+var AWS = require("aws-sdk");
+
+/*Use this for AWS Console*/
+//AWS.config.loadFromPath('../keys.json');
+
+/*Use this for local setup*/
+AWS.config.update({
+  region: "local",
+  endpoint: "http://localhost:8000"
+});
+
+var dynamodb = new AWS.DynamoDB();
+
+var params = {
+    TableName : "V-Transfer",
+    KeySchema: [       
+        { AttributeName: "PK", KeyType: "HASH"},  //Partition key
+        { AttributeName: "SK", KeyType: "RANGE" }  //Sort key
+    ],
+    AttributeDefinitions: [       
+        { AttributeName: "PK", AttributeType: "S" },
+        { AttributeName: "SK", AttributeType: "S" },
+        { AttributeName: "U_ID", AttributeType: "S" },
+        { AttributeName: "LS1_SK", AttributeType: "S"}
+    ],
+    ProvisionedThroughput: {       
+        ReadCapacityUnits: 10, 
+        WriteCapacityUnits: 10
+    },
+    GlobalSecondaryIndexes: [{
+        IndexName: "FIND_FILE",
+        KeySchema: [
+            {
+                AttributeName: "U_ID",
```

Comment:
use GS1_PK as the variable/attribute name, it would make more sense considering the future scope.

Reply:
since their, is a delay in progress. I am merging this pr.
This change will be resolved in next issue.